structcol/montecarlo.py

```diff
@@ -31,7 +31,9 @@
 
 from . import mie, index_ratio, size_parameter
 import numpy as np
+np.random.seed([10])
```

Comment:
Unless you're doing multi-threaded work (where you launch a separate thread for each calculation), it's best to not set a seed value.  In that case, as described in the [numpy manual](https://docs.scipy.org/doc/numpy-dev/reference/generated/numpy.random.RandomState.html):
"If seed is None, then RandomState will try to read data from /dev/urandom (or the Windows analogue) if available or seed from the clock otherwise."